built/authorization_service_configuration.d.ts

```diff
@@ -3,6 +3,7 @@ import { Requestor } from './xhr';
  * Represents AuthorizationServiceConfiguration as a JSON object.
  */
 export interface AuthorizationServiceConfigurationJson {
+    oauthFlowType: string;
```

Comment:
Maintain the attribute consistency eg: oath_flow_type

Reply:
done